### DISCUSSION TITLE
Multiple Config Files

### DISCUSSION THREAD

**Discussion by Spedwards** [Flexget/Flexget#3677] [Q&A]
Hi,

I am thinking about consolidating all my automation stuff down to just Flexget however I don't won't everything clogging up a single config file. I'd ideally like all my uses abstracted to their own config file without any loss in functionality.

This is what my ideal file structure would be:

```
~/.config/flexget/
├─ config.yml
├─ example.yml
├─ movies.yml
├─ podcasts.yml
├─ tv.yml
```

I'm currently only using Flexget for one purpose and my config file is already close to 500 lines long and annoying enough to maintain as is. If I can't abstract my other stuff into their own files (and my current purpose into its own), I won't end up moving everything over.

Is what I'd like to achieve actually possible and if so, how would I go about doing it?

Thanks.

**Comment by gazpachoking** [comment]
You can use the [include plugin](https://flexget.com/Plugins/include) to include task config from multiple files. You still have to list out all your tasks in the main config file, but you can then define the body of them in the separate included config files.
